backend/spec/mixin_touch_records_spec.rb

```diff
@@ -81,4 +81,23 @@
     end
   end
 
+  it 'can update digital object system_mtime value when related ao is updated' do
+    digital_object = create(:json_digital_object, publish: true)
+    digital_object_orig_mt = DigitalObject[digital_object.id].system_mtime
+    opts = {:instance_type => "digital_object",
+      :digital_object => {:ref => digital_object.uri}
+    }
+
+    resource = create(:json_resource)
+    archival_object = create(:json_archival_object,
+                                    :dates => [],
+                                    :resource => {:ref => resource.uri},
+                                    :instances => [build(:json_instance, opts)])
+
+    ArchivesSpaceService.wait(:long)
+    ArchivalObject[archival_object.id].update_from_json(archival_object)
+    digital_object_updated_mt = DigitalObject[digital_object.id].refresh.system_mtime
+    expect(digital_object_updated_mt).to be > digital_object_orig_mt
```

Comment:
This appears to be supported by https://github.com/archivesspace/archivesspace/blob/master/backend/app/model/mixins/relationships.rb#L841, so maybe the changes to `backend/app/model/archival_object.rb` are not actually needed?